source/sass/31-molecules/map/_leaflet.scss

```diff
@@ -65,7 +65,7 @@
 
   .leaflet-control-zoom {
     a {
-      box-shadow: 0 8px 12px 0 rgba(0, 125, 179, .06);
+      box-shadow: 0 3px 14px rgba(0, 0, 0, 0.4);
```

Comment:
If this is a reusable style, maybe interesting to make it available as a mixin?

Reply:
@lennartvava I am not sure how generic this mixin should be
For now, I made it specific to map buttons ( minxin name )  , but let me know if I should make it more generic

Reply:
@lennartvava  @SvenVdv  I made the changes and made the mixin more generic